docker/go-control-plane/README.md

```diff
@@ -0,0 +1,16 @@
+# A simple xDS control panel implementation for tests
+
+## Overview
+
+An xDS control panel implementation that provides special gRPC API for tests
+that require a fine-grained control over the control plane behavior.
+
+## Building
+
+Docker requires that all the files are under the build root and this project
+depends on `protos/grpc/testing/xdsconfig`, the build needs to be ran from
+the root of `grpc/psm-interop` checkout.
+
+```
+docker build . -f docker/go-control-plane/Dockerfile
+```
```

Comment:
Has some go-specific configuration that I did not check in yet as it might be specific to my machine. I found little utility in running this locally, as xDS API is not very accessible for grpcurl. 

All in all - I didn’t need to run it outside Docker and build is really fast. Would like to skip local build for now.